### PR TITLE
[Fix] Fix install bug

### PR DESCRIPTION
**Pull request by zhouzaida**

### CODE REVIEW

File: mim/utils/default.py [hunk 1]
```diff
@@ -23,9 +23,15 @@
     'mmocr': 'mmocr',
     'mmgen': 'mmgeneration',
 }
-PKG_ALIAS = {
+PKG2MODULENAME = {
```
Comment (ZwwWayne) [inline mim/utils/default.py:26]: How about simply using PKG2MODULE and MODULE2PKG

Comment (zhouzaida) [reply mim/utils/default.py:26]: yet, agree.

**Review by ZwwWayne — APPROVED** [review]

File: mim/commands/install.py [hunk 2]
```diff
@@ -401,11 +402,11 @@ def install_from_repo(repo_root: str,
     if is_user_dir:
         install_cmd.append('--user')
 
+    os.environ['MKL_SERVICE_FORCE_INTEL'] = '1'
```
Comment (ZwwWayne) [inline mim/commands/install.py:405]: Check whether this environment variable is necessary. Usually, we do not need that for installing MMCV.

File: mim/commands/search.py [hunk 3]
```diff
@@ -30,6 +31,7 @@
     'batch_size': 'bs',
     'epochs': 'epoch',
     'inference_time': 'fps',
+    'inference_time_(fps)': 'fps',
```
Comment (kennymckormick) [inline mim/commands/search.py:34]: Why both inference_time and inference_time_(fps) maps to fps? Is there any difference between the two names?

File: mim/commands/install.py (hunk 2 above)
Comment (zhouzaida) [reply mim/commands/install.py:405]: reslove issue #12 


Reproduce the bug
```bash
MMCV_WITH_OPS=1
```
```python
import subprocess
import numpy as np
subprocess.check_call(['python', '-m', 'pip', 'install', '/home/username/codebases/mmcv'])
```

**Pull request merged**